.gitignore

```diff
@@ -1,2 +1,2 @@
 id_rsa
-.DS_Store
\ No newline at end of file
+.DS_Store
```

Comment:
Unnecessary changes. Revert it. Reset looks good :+1:

Reply:
Fixed!